src/main/java/com/google/api/codegen/util/go/GoTypeTable.java

```diff
@@ -20,18 +20,26 @@
 import com.google.api.codegen.util.TypeTable;
 
 import java.util.Collections;
-import java.util.List;
 import java.util.Map;
+import java.util.TreeMap;
 
 public class GoTypeTable implements TypeTable {
+
+  private static final String EMPTY_PROTO_PKG = "github.com/golang/protobuf/ptypes/empty";
```

Comment:
This type table, since it is under codegen/util, shouldn't be aware of protobuf-specific things.